components/dashboard/opportunity/index.tsx

```diff
@@ -0,0 +1,20 @@
+import * as S from "./styles";
+
+interface Props {
+  email: string,
+  name: string,
+}
+
+const OwnerContact: React.FC<Props> = ({ email, name }) => {
+  if (email.length > 0) {
```

Comment:
Tohle by mělo být idiomaticky tak, že email bude nepovinná prop:

```typescript
interface Props {
   email?: string,
   name: string,
 }
```

A následně:

```typescript
if (email) { … }
```

Případně koukám, že pole `email` je u typu `PortalUser` povinné, takže to klidně může být povinná prop a můžeš úplně vypustit ten `if`.

Reply:
Já se spíš bojím toho, že to bude prázdný string, to se stát může, ne?